metadata.json

```diff
@@ -57,7 +57,7 @@
     },
     {
       "name": "puppetlabs/concat",
-      "version_requirement": ">= 3.0.0 < 5.0.0"
+      "version_requirement": ">= 5.0.0 < 6.0.0"
```

Comment:
Is 5.0.0 really a minimum requirement?

Reply:
If you try `"version_requirement": ">= 4.0.0 < 5.0.0"`, you will get also an error about unsatisfied dependencies.

Reply:
this should probably be ```">= 3.0.0 < 6.0.0"```

Reply:
@Dan33l have you tried `"version_requirement": ">= 4.0.0 < 6.0.0"`? Let's not bump the minimum version until we have to

Reply:
@dhollinger i didn't noted all my tests. 
The range you proposed looks to work well.
I push this.